components/application-mgt/org.wso2.carbon.identity.application.common/src/main/java/org/wso2/carbon/identity/application/common/constant/AuthenticatorMgtErrorConstants.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.application.common.constant;
+
+/**
+ * Constants for authenticator configuration management service.
+ */
+public class AuthenticatorMgtErrorConstants {
+
+    /**
+     * Error messages.
+     */
+    public enum ErrorMessages {
+
+        // Client errors.
+        ERROR_NOT_FOUND_AUTHENTICATOR("60001", "No Authenticator is found.",
+                "No authenticator is found by given authenticator name: %s."),
+        ERROR_OP_ON_SYSTEM_AUTHENTICATOR("60002", "No operations allowed on system authenticators.",
+                "Do not allow to perform any operation on system defined authenticator: %s."),
+        ERROR_AUTHENTICATOR_ALREADY_EXIST("60003", "There is already an authenticator.",
+                "There is already an authenticator by the given name: %s."),
+        ERROR_INVALID_AUTHENTICATOR_NAME("60004", "Invalid authenticator name is provided.",
+                "The provided authenticator name %s is not in the expected format %s."),
+        ERROR_BLANK_FIELD_VALUE("60004", "Blank field value is provided.",
```

Comment:
Can we say "Invalid empty value" and improve the description to "Value for %s should not be empty."

Reply:
be consistent on the term use either blank or empty

Reply:
Addressed with commit 86c5ebf1673fdcb0e0cb51a4b032a54b86b88242